wcivf/apps/people/templates/people/includes/_person_previous_elections_card.html

```diff
@@ -29,8 +32,18 @@ <h3 class="ds-padded">{% trans "Previous Elections" %}</h3>
                         {% else %}
                             <td>{% if person_post.votes_cast %}{% blocktrans with num_votes=person_post.votes_cast|intcomma %}{{ num_votes }} votes (not elected){% endblocktrans %}{% else %}{% trans "Vote count not available" %}{% endif %}</td>
                         {% endif %}
+                        {% if object.previous_party_count %}
+                            <td>
```

Comment:
You dont want another loop through the candidacies here, as in this block it is already looping through `PersonPost` (candidacies) objects - see line 21. So we want to loop through the `previous_party_affiliations` for the `person_post`